src/components/PreviewFrame.jsx

```diff
@@ -85,17 +85,15 @@ class PreviewFrame extends React.Component {
   }
 
   _runtimeErrorLineOffset() {
-    const firstSourceLine = this.props.compiledProject.source.
-      split('\n').indexOf(sourceDelimiter) + 2;
-
-    return firstSourceLine - 1;
+    return this.props.compiledProject.source.
+      split('\n').findIndex(l => l.indexOf(sourceDelimiter) !== -1) + 1;
```

Comment:
Still using the delimiter because the line numbers come from `outerHTML` instead of from source directly

Reply:
Just to make sure I’m understanding—the notion here is that us having a sourcemap doesn’t change the browser’s behavior with respect to line numbers in the error, which is who-knows-what since this code is in a `<script>` tag, right? What was good for the goose is still good for the gander?